spec/support/city_macros.rb

```diff
@@ -0,0 +1,30 @@
+module CityMacros
+
+  def create_city(city = build(:city))
+    navigate_cities_index
+
+    click_link 'New City'
+    fill_in 'Name', with: city.name
+    click_button 'Create City'
+  end
+
+  def update_city(city = build(:city))
+    navigate_cities_index
+    click_link('Edit', match: :first)
+    fill_in 'Name', with: city.name
+    click_button 'Update City'
+  end
+
+  def destroy_city
```

Comment:
Changes have been added in [cd84a0b](https://github.com/andru1989/population/commit/cd84a0b5e5865c2e69f9db223be960c033e10a28)